frontend/src/Components/StoreSearch/StoreCard.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from "react";
+import { Card, Button, Badge } from "react-bootstrap";
+
+const getOpenString = (hours) => {
+    const currentHour = new Date().getHours();
+    if (currentHour < hours[0] || currentHour > hours[1]) {
+        return [false, ("Opens at " + hours[0] + ":00")];
+    }
+    else {
+        return [true, ("Open until " + hours[1] + ":00")];
```

Comment:
Also something we don't have to address in this PR, but to keep in mind for the future: Do we want to display the time in 1-12 format or 0-23?

Reply:
Yeah this is definitely on the to do list. I didn't feel like writing more robust conditionals for the prototype of this page.